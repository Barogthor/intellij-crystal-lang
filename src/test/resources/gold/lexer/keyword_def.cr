def size
  items.size
end