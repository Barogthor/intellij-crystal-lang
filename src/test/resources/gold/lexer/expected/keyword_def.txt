def ('def')
whitespace (' ')
identifier ('size')
whitespace ('\n  ')
identifier ('items')
. ('.')
identifier ('size')
whitespace ('\n')
end ('end')